bftengine/src/bftengine/ValidationOnlyIdentityManager.cpp

```diff
@@ -0,0 +1,53 @@
+// Concord
+//
+// Copyright (c) 2020-2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+#include "ValidationOnlyIdentityManager.hpp"
+
+namespace bftEngine::impl {
+
+ValidationOnlyIdentityManager::ValidationOnlyIdentityManager(
+    PrincipalId myId,
+    const std::vector<std::pair<Key, concord::crypto::KeyFormat>>& publickeys,
+    const std::map<PrincipalId, KeyIndex>& publicKeysMapping,
+    const ReplicasInfo& replicasInfo)
+    : SigManager(myId,
+                 {"", concord::crypto::KeyFormat::HexaDecimalStrippedFormat},
+                 publickeys,
+                 publicKeysMapping,
+                 false,
+                 {},
+                 replicasInfo) {}
+
+bool ValidationOnlyIdentityManager::verifySig(
+    PrincipalId pid, const concord::Byte* data, size_t dataLength, const concord::Byte* sig, uint16_t sigLength) const {
+  return verifyNonReplicaSig(pid, data, dataLength, sig, sigLength);
```

Comment:
If it validates only NonReplicaSigs, please specify it in the method name

Reply:
Can't, need to override the parent's method.